### PR TITLE
add client.sh and update readme.md

### PR DESCRIPTION
**Pull request by d78ui98**
- [x] added `client.sh` for creating ltsp clients
- [x] updated `README.md` for `buster` branch

### CODE REVIEW

**Comment by d78ui98** [comment]
I added usage function in `client.sh`. Also reduced its complexity. If this is good I can create a pull request to this in `bionic` branch too.

File: client.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	
+	echo -n "Enter any name for LTSP client : "
+	read clientname
+	echo "Using ${INTERFACE}.."	
+	VBoxManage createvm --name "${clientname}" --register
+	cd "$HOME/VirtualBox VMs/${clientname}"
+	VBoxManage modifyvm "${clientname}" --memory 1024 --acpi on\
+ --boot1 net --boot2 none --boot3 none --boot4 none --nic1 bridged\
+ --bridgeadapter1 eno1 --nicpromisc1 allow-all
+	VBoxManage createhd --filename "$HOME/VirtualBox VMs/${clientname}/${clientname}.vdi" --size 1
```
Comment (dashohoxha) [inline client.sh:41]: Indentation is not Ok. I don't understand how you can commit such a file, after me asking you to correct indentation so many times. A mistake is Ok, a repeatedly repeated mistake is not Ok.
Please use some editor that can do the indentation correctly. If nothing else, spend 2-3 days to get familiar with Emacs.

**Comment by dashohoxha** [comment]
> If this is good I can create a pull request to this in bionic branch too.

I don't know it it is good, because you have not tested it. You cannot test it because the server does not work in `buster`. I will accept it anyway, but you should do it correctly for `bionic`, testing it as well,  and then come back and fix it for `buster`.

**Comment by dashohoxha** [comment]
Also try to keep pull requests and commits meaningful. For example adding the code of the client and updating README have no relation to each-other, so they should have been (ideally) separate pull requests, or at least separate commits.

**Pull request merged**